pkg/controllers/update.go

```diff
@@ -64,7 +64,7 @@ func RunInstallScript() (bool, string, Error) {
 	command := []string{tmpFile, "--debug"}
 
 	startTime = time.Now()
-	out, err := exec.Command(command[0], command[1:]...).CombinedOutput() // #nosec G204
+	out, err := exec.Command("sh", command...).CombinedOutput() // #nosec G204
```

Comment:
Could you pull this into a separate commit? Also, could you provide some more info on why this changed is needed?

Reply:
I take that back. Using just `env` does work as expected. I had forgotten that I'd not already installed the CLI in my fresh Windows 11 install, so it was erroring out with a 127 since it wasn't finding something to update (the dev build of the CLI wasn't on the PATH).

Do we want to go that route? It seems unlikely that `sh` wouldn't be on the PATH already if its installed on a system (and it definitely is in Git Bash).

Reply:
This is difficult to say without testing this command thoroughly. I'm wondering if we can only run this alternative command when on Windows?

Reply:
We could probably do that easily enough. Shall I go that route?

Reply:
Latest force-push addresses this. [compare](https://github.com/DopplerHQ/cli/compare/71bc30575f0e8e23b3bf83bcdb10b5f70a3c6eb3..4d904c7c2777173359da987e4385909e5487117c)